tests/test_service.py

```diff
@@ -45,7 +45,7 @@ def mosec_service(request):
         shlex.split(f"python -u tests/{name}.py {args} --port {TEST_PORT}"),
     )
     assert wait_for_port_open(port=TEST_PORT), "service failed to start"
-    yield service
+    yield (service, args)
```

Comment:
Will this change affect other test cases?

Reply:
No, Since our tests use HTTP requests to communicate, the popen instance `service` hasn't been used yet.

Reply:
Better not do that. Since you can control the input to this fixture, you don't have to get the same input from this yield.